docs/decisions/0009-webhook-based-communication.rst

```diff
@@ -0,0 +1,197 @@
+################################################
+Webhook-Based Fulfillment Service Communication
+################################################
+
+Status
+******
+
+**Draft**
+
+Context
+*******
+
+We are designing a new fulfillment engine as a Django-based service (Order Fulfillment, or OF) to process fulfillment (e.g., fulfilling in LMS) requests initiated by the Commerce Coordinator (CC) service. These services will be maintained by different organizations — OF by 2U, and CC by Red Ventures.
```

Comment:
As I understand it today, CC is not going to be maintained by Red Ventures. Any functionality from "commerce-coordinator-esque" that RV will want to maintain will be recreated on the RV side.

This is due to operational reasons: RV employees cannot gain VPN access presently, and so cannot deploy or troubleshoot commerce-coordinator in depth, as well as RV employees cannot (should not) run devstack, as there is far too much edx specific context required to run this tool when most of the work we'll be doing is no longer in that ecosystem.

The future is looking like some endpoint will exist on 2U's side (probably the fulfillment service), and something on the RV side will call it after we accept payment upstream.

Reply:
I don't think adding addition information about the intended future state of CC is needed for this ADR.

This ADR covers only the splitting out of the OF service, and I think it accurately states that once the OF service is split out, ownership will transfer to 2U, and CC (as long as it continues to exist) will continue to be maintained by RV.